WordPress/src/main/java/org/wordpress/android/ui/main/SitePickerAdapter.java

```diff
@@ -276,6 +291,50 @@ public boolean onLongClick(View view) {
                 }
             });
         }
+
+        if (mIsSingleItemSelectionEnabled) {
+            if(getSitesCount() == 1){
```

Comment:
I know we're basically handling the `0` sites case elsewhere (by not showing the Site picker at all) but I would feel safer if we check it here as well.

For example, let's check for `<= 1` than just `== 1` in order to hide the checkbox. WDYT?

Reply:
The code is in the onBindViewHolder which can't be called when there is no item in the adapter. WDYT @hypest

Reply:
> ...which can't be called when there is no item in the adapter.

True, but i'm referring to the zero *sites* case which is different to the zero *items* case since there might be a header item.

The assumption here is that execution won't reach this point if `getSitesCount() == 0` but that's not apparent while reading the code so, we should either handle that case as commented [originally](https://github.com/wordpress-mobile/WordPress-Android/pull/6690#discussion_r144536607) or we should add a comment that we're expecting never to reach this line when no sites are available.

Reply:
Ohh I see your point. Fixed:)